src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -40,10 +40,10 @@ const ReportActionItemCreated = (props) => {
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
     return (
         <OfflineWithFeedback
-            pendingAction={lodashGet(props.report, 'pendingFields.addWorkspaceRoom')}
-            errors={lodashGet(props.report, 'errorFields.addWorkspaceRoom')}
+            pendingAction={lodashGet(props.report, 'pendingFields.addWorkspaceRoom') || lodashGet(props.report, 'pendingFields.createChat')}
+            errors={lodashGet(props.report, 'errorFields.addWorkspaceRoom') || lodashGet(props.report, 'errorFields.createChat')}
```

Comment:
Same here we need to combine these into one error object.

Reply:
I'm not sure I need to add the same comment here. Let's discuss in the above thread before I change anything here.